src/Card/Card.tsx

```diff
@@ -46,14 +49,15 @@ export const Card: FC<CardProps> = ({
         {...attributes}
         className={cardClasses}
         tag={tag}
+        href={href}
```

Comment:
try this if work    ...(tag === ''a && { href: href }),

Reply:
That doesn't look like correct syntax to me, but maybe I did not understand where to put it. Also, it would return `false` if `tag !== 'a'` or the object `{href: href}` if `tag === 'a'`

Reply:
i don't know if it would work, i usually use if to add the object property only if it satisfies the condition

Reply:
From the research I made, I think there's no way to do what you want to do, but if you find a way feel free to push a commit. If you are a maintainer you have the right to do it, otherwise I can give you the needed permission

Reply:
Ok, I put a ternary operator to set `href` to `undefined` when the tag is not an anchor. This way the property won't even be visible among DOM's node' attributes